### DISCUSSION TITLE
LDM Notes for October 12th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6544] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-10-12.md

## Agenda

- Keywordness of `field`

**Comment by huoyaoyuan** [comment]
I'd somehow like the approach to add a modifier to property (just like `async`).
The modifier can be a clear indicator whether it has a backing field.

**Reply by iam3yal** [reply]
The extra modifier may make it crystal clear but then I think that unlike `async` in the long run it will be an annoyance and people gonna ask/wonder "why the extra modifier?" just like they do for `async` only for a feature that is a lot smaller and the answer _might_ be more absurd so I'd say if they can really avoid it they should.

**Reply by BreyerW** [reply]
also `field` is supposed to reduce as much boilerplate as possible. adding modifier goes against that goal so its better to avoid if possible.
